src/stories/Modal.stories.tsx

```diff
@@ -29,47 +19,27 @@ function Template2(args: any) {
       <Modal
         open={open}
         onClose={() => setOpen(false)}
-        {...args}
-      />
+      >
+        <ModalHeader onClose={() => setOpen(false)}>
```

Comment:
what value does splitting out this component provide?

Reply:
More control over the customization of the modal

Reply:
Also, we already have a separate `ModalActions`, so it's in the continuation.

Reply:
Will it break all existing code?  Or is the title prop and onClose still supported on modal itself